pkg/upstream/replicated.go

```diff
@@ -428,13 +428,16 @@ func listPendingChannelReleases(replicatedUpstream *ReplicatedUpstream, license
 		hostname = fmt.Sprintf("%s:%s", u.Hostname(), u.Port())
 	}
 
-	sequence := cursor.Cursor
-	if license.Spec.ChannelName != cursor.ChannelName {
+	sequence := currentCursor.Cursor
+	if license.Spec.ChannelName != currentCursor.ChannelName {
 		sequence = ""
 	}
 
 	urlValues := url.Values{}
 	urlValues.Set("channelSequence", sequence)
+	urlValues.Add("channelName", currentCursor.ChannelName)
```

Comment:
should we get this from the license instead? in case the customer synced the license before checking for updates, this could be different.

Reply:
i think this should come from license.  this would make `channelName ` consistent with `channelSequence` I think